assets-src/styles/sass/30-base/_lists.scss

```diff
@@ -55,6 +55,10 @@ dl {
 		font-weight: bold;
 	}
 
+    dd {
+        margin-inline-start: rem(40);
+    }
```

Comment:
If you try on https://design-system.w3.org/code/event.html you will see that it is also going to add some space between each event meta (e.g. between "Date:") and their value.

Before:
![image](https://github.com/user-attachments/assets/a1dbd96b-589e-4a44-bb12-75ac7b5d7346)

After:
![image](https://github.com/user-attachments/assets/1034762d-9e03-4f0f-8c97-f2edad11c31d)

Reply:
Apologies for not spotting that - I have pushed a fix for it.

Reply:
Thanks. I asked @jean-gui to check this one further so see if there are any other side effects.